src/App.css

```diff
@@ -0,0 +1,108 @@
+header {
+  display: flex;
+  align-items: baseline;
+  width: 30%;
+  justify-content: space-between;
+  color: #0290ff;
+  padding-left: 4%;
+}
+
+a {
+  text-decoration: none;
+  padding: 10%;
+  color: #121212;
+}
+
+main {
+  background: #e8e8e8;
+  padding: 3%;
+}
+
+.book {
+  background: #fff;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 2%;
+  margin-bottom: 1%;
+}
+
+.title {
+  font-size: 1.375rem;
+  font-weight: bold;
+}
+
+.author {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
```

Comment:
- [ ] It is a requirement that your styling is exactly the same as the one provided on Zeplin. Your design happens to need some more styling to meet the requirement. Kindly fix that to have your project approved.

Your design | Zeplin Design
--|--
![image](https://user-images.githubusercontent.com/29071984/187049893-33aad0bf-c7dc-47d5-8fbc-abaf94212dc9.png) | ![image](https://user-images.githubusercontent.com/29071984/187049859-cfb20545-c9ad-4b8b-9a7a-75c2e7c19fa1.png)

Reply:
- [ ] **[REQUIRED]** I noticed that you did not apply the changes requested by the previous reviewer. Your app design is a bit far from the Bookstore design in [Zeplin](https://zeplin.io/). :cry:  Please make sure that your app design is Styling EXACTLY like Zeplin Bookstore's design, use the same styling element provided. As a developer, it's very important to peer attention to details cause your future employer will judge your work on it. And following the zeplin design will make your application works fine and wonderful :smile: 
Please check the screenshots below to see the differences :arrow_down: 

#### Your design
![Screenshot from 2022-08-27 22-50-43](https://user-images.githubusercontent.com/6615394/187050716-8d5f6ead-7c18-440f-8a31-82d7b6e2ea2e.png)



#### Zeplin design
![Screenshot from 2022-08-27 12-28-29](https://user-images.githubusercontent.com/6615394/187030638-2505a940-ea42-41eb-aed4-c4c1939fbfd0.png)
![Screenshot from 2022-08-27 12-28-41](https://user-images.githubusercontent.com/6615394/187030637-667720ed-3fc8-4a73-9fb6-0d646f392918.png)